packages/lesswrong/components/users/UsersProfile.tsx

```diff
@@ -489,6 +503,9 @@ const UsersProfileFn = ({terms, slug, classes}: {
           <div className={classes.centerColumnWrapper}>
           {/* Bio Section */}
           <SingleColumnSection>
+              <div className={classes.nameAndProfileWrapper}>
```

Comment:
Do format this code a bit. I know we don't have an autoformatter, but we do try.